harvard-theologisches-seminar-adelshofen.csl

```diff
@@ -0,0 +1,264 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de-DE">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Harvard - Theologisches Seminar Adelshofen (German)</title>
+    <title-short>TSA</title-short>
+    <id>http://www.zotero.org/styles/harvard-theologisches-seminar-adelshofen</id>
+    <link href="http://www.zotero.org/styles/harvard-theologisches-seminar-adelshofen" rel="self"/>
+    <link href="http://www.zotero.org/styles/harvard-gesellschaft-fur-bildung-und-forschung-in-europa" rel="template"/>
+    <link href="http://www.lza.de/" rel="documentation"/>
```

Comment:
@eugeis, is the style guide available online? Otherwise, http://www.lza.de/theologisches-seminar/ is probably a better documentation URL.